database/build.gradle

```diff
@@ -27,6 +27,10 @@ dependencies {
     // Needed to override play services
     compile "com.android.support:support-v4:$supportLibraryVersion"
 
+    compile 'android.arch.lifecycle:runtime:1.0.0-alpha5'
+    compile 'android.arch.lifecycle:extensions:1.0.0-alpha5'
+    annotationProcessor 'android.arch.lifecycle:compiler:1.0.0-alpha5'
```

Comment:
I wanted to make these `provided` so devs didn't have to pull them in, but I kept getting compile errors for the `database` module so I gave up. [Their](http://www.methodscount.com/?lib=android.arch.lifecycle%3Aruntime%3A1.0.0-alpha5) [size](http://www.methodscount.com/?lib=android.arch.lifecycle%3Aextensions%3A1.0.0-alpha5) isn't too big though.